gym_xiangqi/envs/xiangqi_env.py

```diff
@@ -153,6 +153,36 @@ def get_possible_actions(self):
         """
         Searches all valid actions each piece can perform
         """
+        # Clear all possible actions to remove possible actions from
+        # previous turn.
+        # TODO: Don't clear the entire list, but only the relevant actions.
+        self.possible_actions.fill(0)
```

Comment:
@hojoung97, this is a temporary fix. Feel free to remove it once you implemented the `step` method. I think it should solve the problem I was seeing.

Reply:
sounds good 👍

Reply:
sounds good 👍